recipes/zimg/all/conanfile.py

```diff
@@ -0,0 +1,136 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, MSBuild, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+
+
+class ZimgConan(ConanFile):
+    name = "zimg"
+    description = "Scaling, colorspace conversion, and dithering library"
+    topics = ("conan", "zimg", "image", "manipulation")
+    homepage = "https://github.com/sekrit-twc/zimg"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "WTFPL"
```

Comment:
hilarious! almost better than Beerware!

Reply:
Maybe that is funny between friends, but from a legal standpoint such "licenses" are highly problematic 🤕

Reply:
@Croydon Totally!!